### ISSUE TITLE
Support Exporting of JUNO (IBC token) to Ethereum

### ISSUE THREAD

**Issue by pandaring2you** [Sifchain/sifnode#1998]
Hey @daniportugalgit - have you supported this before? Wasn't sure if you or @banshee  was the right person

**Comment by daniportugalgit** [comment]
Yes, this is something I've done before, together with James and Elliot. I'll talk to @banshee to remember the details.

**Comment by pandaring2you** [comment]
@caseyturnerarrington

**Comment by daniportugalgit** [comment]
I'm reviewing and refactoring the scripts to do this in a more streamlined way.

**Comment by caseyturnerarrington** [comment]
Awesome thanks Jordan. Can we add DVPN as well (Sentinel’s token)? They want to get on QS and DS ASAP!

**Comment by pandaring2you** [comment]
cc @yvesmessy

**Comment by musnit** [comment]
> Hey @musnit - can you run https://github.com/Sifchain/sifnode/blob/705d849a4922898fde0469a1516558295ce9867b/smart-contracts/RegistgerIbcTokens.md as @daniportugalgit mentioned above?
> 
> This step should register all the above tokens on bridgebank and would allow users to export DVPN and JUNO to ethereum, but they would still not be able to transfer the tokens to Polygon until the polygon team confirms they have bridged the tokens.

code and scripts look good. @intl-man can you execute?
